include/PackedFloat.h

```diff
@@ -149,6 +172,19 @@ class PackedFloat {
         return !(*this == rhs);
     }
 
+    /// Slightly looser comparison where zero values are allowed to differ in the sign and exponent
+    inline bool EquivCompare(PackedFloat const &rhs) {
```

Comment:
I would be in favor of adding this to the regular equals operator. Is there a case where we want the stricter compare?

Reply:
It seemed like you were going for exact bit compatibility with MPFR. This comparison relaxes that slightly for 0

Reply:
I would like to follow MPFR semantics as closely as possible, so if this is how they treat zero, and it's no significant extra work for us to do the same, I think we should too

Reply:
I think it's a fair bit of work to get exact bit compatibility of the mantissa at zero with MPFR. What would be reasonable is encoding the singular values + zero in the exponent like MPFR does. For context, currently we are encoding that zero mantissa = zero

Reply:
Per our convo, we'll circle back to this later